### PR TITLE
Fix the build problem on Ubuntu

### PR DESCRIPTION
**Pull request by sherman-the-tank**

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

File: project/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,12 @@
-cmake_minimum_required(VERSION 3.8.0)
-cmake_policy(SET CMP0114 OLD)
+cmake_minimum_required(VERSION 3.12.0)
```
Comment (jiayuehua) [inline project/CMakeLists.txt:1]: should be 3.13 simdjson need 3.13

**Review by wenhaocs — DISMISSED** [review]

File: project/externals/wangle.cmake [hunk 2]
```diff
@@ -35,8 +35,8 @@ ExternalProject_Add_Step(${name} mannual-configure
         -DBUILD_TESTS=OFF
         -DCMAKE_EXE_LINKER_FLAGS=-latomic
         -DCMAKE_SHARED_LINKER_FLAGS=-latomic
-        -D_OPENSSL_LIBDIR=${CMAKE_INSTALL_PREFIX}/lib64
-        .
+	-D_OPENSSL_LIBDIR=${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
```
Comment (jiayuehua) [inline project/externals/wangle.cmake:38]: using whitespace

File: project/externals/cachelib.cmake [hunk 3]
```diff
@@ -9,6 +9,7 @@ ExternalProject_Add(
     ${name}
     GIT_REPOSITORY https://github.com/facebook/CacheLib.git
     GIT_TAG 12822a2eddd04bb3e252f458b36ee0675a2d4643  # As of 2021/12/02
+    GIT_SUBMODULES ""
```
Comment (jiayuehua) [inline project/externals/cachelib.cmake:12]: why add this line?

**Review by jiayuehua — DISMISSED** [review]

File: project/CMakeLists.txt (hunk 1 above)
Comment (sherman-the-tank) [reply project/CMakeLists.txt:1]: Good catch. Actually the version of simdjson in third-party requires 3.14

I'll change it

File: project/externals/wangle.cmake (hunk 2 above)
Comment (sherman-the-tank) [reply project/externals/wangle.cmake:38]: Good catch :+1:

File: project/externals/cachelib.cmake (hunk 3 above)
Comment (sherman-the-tank) [reply project/externals/cachelib.cmake:12]: This empty string combined with policy CMP0097 will avoid pulling all sub-modules. We don't use the libraries from the sub-modules anyway

**Review by jiayuehua — APPROVED** [review]

**Pull request merged**